### DISCUSSION TITLE
Why can‘t I use the mobile version of github to create new files?

### DISCUSSION THREAD

**Discussion by MTpupil** [community/community#70156] [Mobile] [answered]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, 版本 1.100.0 (91333308)
Device: iPad Mini 6 (iOS 16.1)
</details>

**Comment by sunny0625** [comment]
Can You share the screenshot or a sample video of your issue?

**Comment by esahmed21** [comment, marked as answer]
Hi @MTpupil 

Thank you for reaching out to us!

Would you please update the App to a latest version so you can use the create files feature. I see you're on v1.100.0 (91333308) which is old and doesn't have that feature.

Thank you!

**Reply by MTpupil** [reply]
Thank you for your answer. I just noticed that I haven't updated it for a long time.
